src/test/java/edu/ucla/library/bucketeer/handlers/DeleteJobHandlerTest.java

```diff
@@ -58,7 +58,8 @@ public final void testDeleteJobHandler(final TestContext aContext) {
                                 final HttpResponse<Buffer> response = deletion.result();
                                 final int statusCode = response.statusCode();
 
-                                aContext.assertEquals(HTTP.NO_CONTENT, statusCode, LOGGER.getMessage(
+                                //aContext.assertEquals(HTTP.NO_CONTENT, statusCode, LOGGER.getMessage(
```

Comment:
Did you mean to leave this commented out text in?

Reply:
I was experimenting, will remove the commented line before making this an official PR